qemu/tests/win_sigverif.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import re
+
+from avocado.core import exceptions
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+from virttest.utils_windows import system
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    sigverif test:
+    1) Boot guest with related virtio devices
+    2) Run sigverif command(an autoit script) in guest
+    3) Open sigverif log and check whether driver is signed
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    driver_name = params["driver_name"]
+    session = utils_test.qemu.windrv_check_running_verifier(session, vm,
+                                                            test, driver_name)
+
+    run_sigverif_cmd = utils_misc.set_winutils_letter(
+        session, params["run_sigverif_cmd"])
+    sigverif_log = params.get("sigverif_log")
+    check_sigverif_cmd = params.get("check_sigverif_cmd") % driver_name
+    clean_sigverif_cmd = params.get("clean_sigverif_cmd")
+
+    error_context.context("Run sigverif in windows guest", logging.info)
+    status, output = session.cmd_status_output(run_sigverif_cmd)
+    if status != 0:
+        raise exceptions.TestError(output)
+
+    if not utils_misc.wait_for(lambda: system.file_existing(session,
+                               sigverif_log), 180, 0, 5):
+        test.error("sigverif logs are not created")
+
+    try:
+        error_context.context("Open sigverif logs and check driver signature"
+                              " status", logging.info)
+        output = session.cmd_output(check_sigverif_cmd)
+        if re.search("Not Signed", output):
```

Comment:
Better enable multi line check(re.M), for viorng device, the output will include multi lines.
viorngci.dll             12/12/2018     ��[                  Signed              viorng.cat          Red Hat, Inc.
viorngum.dll             12/12/2018     ��[                  Signed              viorng.cat          Red Hat, Inc.
viorng.sys               12/12/2018     ��[                  Signed              viorng.cat          Red Hat, Inc.

suggest change to  if re.search("Not Signed", output, re.M):

Reply:
Updated, thanks